### ISSUE TITLE
g:mt_IDontWantTemplatesAutomaticallyInserted value not being retaining value from .vimrc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/lh-vim#30]
```
What steps will reproduce the problem?
1. set let g:mt_IDontWantTemplatesAutomaticallyInserted = 1 in .vimrc
2. open new vim instance
3. echo g:mt_IDontWantTemplatesAutomaticallyInserted

What is the expected output?
1

What do you see instead?
0

What version of the product are you using? On what operating system?
2.1.0 on WinXP

Please provide any additional information below.
commenting out the following line in after/plugin/mu-template.vim fixes the 
issue: 1047.
call lh#menu#def_toggle_item(s:AutoInsertMenu)
```

Original issue reported on code.google.com by `peter.ri...@gmail.com` on 9 Dec 2010 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
Indeed.
This is an issue with lh#menu#def_toggle_item() that ignores pre-existing 
values.
```

Original comment by `luc.herm...@gmail.com` on 9 Dec 2010 at 6:04
- Changed state: **Accepted**
- Added labels: **lhVimLib**, **muTemplate**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision r303, mu-template version 2.1.1
```

Original comment by `luc.herm...@gmail.com` on 9 Dec 2010 at 11:37
- Changed state: **Fixed**
- Removed labels: **lhVimLib**
